### DISCUSSION TITLE
NFTs for the Golden Soroban Story

### DISCUSSION THREAD

**Discussion by Ninotschka69** [stellar/sorobanathon#145] [General]
Here are my NFTs for the Sorobanathon 

 -> The Quest has begun 
  
[https://litemint.com/items/GCTDXFBLBNC7IUACM3XWKGZGJXMUWQD7O4YEFUTJRDYIZSGYOJDVRAMI/sorochap0](url)

-The Door to Squaktahune Temple - Chapter 1

 [https://litemint.com/items/GCKXYN2DFQ5KJJTS4U4YJLDKNDLRTXXTKBAO5C7OLZ4C2DLZISAFVWL3/sorochap01](url)

-The Entrance Hall Pit - Chapter 2

[https://litemint.com/items/GB7DQKEH3YSU2GDYDYLN7QS4OABR2UNZLILVE2WMM2JR53PY3XTX5RVR/sorochap02](url)

-Ring of Fire Riddle - Chapter 3

[https://litemint.com/items/GAXL56GJXSYTYXFDNOO3TD3547SJYWNOC3IQ6ZHCXCWVABAL7R4BCPXD/sorochap03](url)

-The Keeper of Lore - Chapter 4

[https://litemint.com/items/GBNP3VWQT5BNKEW2AIGLJH5GA3F45VF7GQQQ6YXW4H2FXUUH2XCLDL5Z/sorochap04](url)
  
-Zebulons Chance - Chapter 5

[https://litemint.com/items/GAMLJJV22REOOLDVXCSOVMR4GEWM5QGH2YZHE7XXV7OPHQUDJPRWS6SG/sorochap05](url)

-Riding the Luck  - Chapter 6

[https://litemint.com/items/GDKDNWKBE24MC5W33WJEJ2N2Y2KX4AVJHJ24E62UDLZFRJ5EVDI5D2XL/sorochap06](url)

thx and merry christmas y'all

**Comment by stellarjohn** [comment]
Hi [Ninotschka69](https://github.com/Ninotschka69)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by loveway** [comment]
您的邮件已收到，谢谢。                                  -----Loveway丶

**Comment by britzlieg** [comment]
您好，你的邮件我已经收到~
